### PR TITLE
s/Travis CI/GitHub Actions

### PR DESCRIPTION
**Pull request by trentm**
Refs: https://github.com/elastic/apm-agent-nodejs/issues/1922

### CODE REVIEW

**Comment by trentm** [comment]
You can see the badge updates on the README here: https://github.com/elastic/object-filter-sequence/tree/trentm/travis-bye/

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+jobs:
+  test-vers:
+    strategy:
+      matrix:
+        node: ['8.6.0', '8', '10', '12', '14']
```
Comment (astorm) [inline .github/workflows/test.yml]: why both 8.6 and 8?

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "precoverage": "npm run test",
     "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
-    "pretest": "standard",
+    "lint": "standard",
     "test": "nyc tape test.js"
```
Comment (astorm) [inline package.json:10]: 👍 changing this to the more standard "lint"

File: .github/workflows/test.yml (hunk 1 above)
Comment (astorm) [inline .github/workflows/test.yml]: Also -- just noting that we've dropped the Node 6 support that was in the old travis file -- this is appropriate.

**Review by astorm — APPROVED** [review]
Approving. Nothing out of line here -- just some out of curiosity questions below.

File: .github/workflows/test.yml (hunk 1 above)
Comment (trentm) [reply .github/workflows/test.yml]: Copying apm-agent-nodejs in that somewhat: test the latest of each supported release stream, plus the min ver (v8.6.0)

I suppose the travis.yml I'm deleting there included version 6, and it looks like GH Actions still supports a v6 version: https://github.com/actions/node-versions/blob/main/versions-manifest.json#L3278

Comment (trentm) [reply .github/workflows/test.yml]: I guess. A drop of a node major version should, IMHO, be a major version bump. I'll re-add node 6 here for testing.

**Comment by trentm** [comment]
Maybe silly to keep including 8.6.0 here. We can handle that odd case in apm-agent-nodejs.

File: .github/workflows/test.yml (hunk 1 above)
Comment (trentm) [reply .github/workflows/test.yml]: I've dropped the 8.6.0 special -> testing that min ver belongs in apm-agent-nodejs.git.

**Pull request merged**